src/components/UI/dropdown/dropdown.module.scss

```diff
@@ -0,0 +1,143 @@
+// scss variables
+$border-color: #000000;
+$black: #000000;
+$text-color: #ffffff;
+$hover-color: #49a82e;
+$background-color: #ffffff;
+
+.userGreet {
+  color: $text-color;
+  margin: 23px;
+}
+
+.userGreet a {
+  padding: 0px;
+}
+
+.userProfilePic {
+  width: 32px;
+  height: 32px;
+  border-radius: 50%;
+  display: inline;
+  vertical-align: middle;
+}
+.dropdownHeader {
+  margin: 10px;
+  cursor: pointer;
+}
+
+.userGreetMsg {
+  display: inline;
+  vertical-align: middle;
+  margin-right: 10px;
+  color: rgb(255, 255, 255);
+}
+
+.userGreetMsg:hover {
+  color: $hover-color;
+}
+
+.myProfileWrapper {
+  padding-bottom: 15px;
+  text-align: center;
+  border-bottom: 1px solid $border-color;
+}
+
+.myProfile:hover {
+  color: $hover-color;
+}
+
+.myProfile {
+  color: $black;
+  text-decoration: none;
+}
+
+.signOutWrapper {
+  padding-top: 15px;
+  text-align: center;
+  border-top: 1px solid $border-color;
+}
+
+.signOutWrapper > button {
+  all: unset;
+  cursor: pointer;
+}
+
+.signOutWrapper button:focus-visible {
+  outline: var(--black) 5px auto;
+}
+
+.signOut:hover {
+  color: $hover-color;
+  cursor: pointer;
+}
+
+.dropdownContent {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: $black;
+  border: 2px solid $border-color;
+  height: 100%;
+  border-radius: 8px;
+  width: 100%;
+  justify-items: center;
+}
+
+.dropDownContentWrapper {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.dropDownContentWrapper div {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.dropDown {
+  background-color: $background-color;
+  width: 180px;
+  position: absolute;
+  overflow: hidden;
+  right: 35px;
+  border-radius: 8px;
+  box-shadow: 0 10px 25px rgba(0, 0, 0, 0.1);
```

Comment:
same as above, why are we using RGB here?

Reply:
In the previous code rgb was used many times, so from taking Idea fom that I added the RGB, please let me know what changes can I do.